src/utilities/Animate.ts

```diff
@@ -41,8 +41,31 @@ namespace fabric {
   
   export class Animate {
     
-    private static _transformProps: Array<string> = ["x", "y", "z", "scaleX", "scaleY", "scaleZ", "rotate", "rotateX", "rotateY", "rotateZ", "skewX", "skewY"];
-	  private static _filters: Array<string> = ["blur", "brightness", "contrast", "dropShadow", "grayscale", "hueRotate", "invert", "saturate", "sepia"];
+    private static _transformProps: Array<string> = [
+      "x", 
+      "y", 
+      "z", 
+      "scaleX", 
+      "scaleY", 
+      "scaleZ", 
+      "rotate", 
+      "rotateX", 
+      "rotateY", 
+      "rotateZ", 
+      "skewX", 
+      "skewY"
+    ];
+	  private static _filters: Array<string> = [
```

Comment:
The tabbing for these two arrays could be normalized and aligned w/the statics below